src/main/java/com/ayuconpon/userCoupon/domain/UserCouponRepository.java

```diff
@@ -1,11 +1,20 @@
 package com.ayuconpon.userCoupon.domain;
```

Comment:
```suggestion
package com.ayuconpon.usercoupon.domain;
```

@cire0304 님, 보통 패키지명은 소문자로 하는것을 컨벤션으로 합니다.

Reply:
컨밴션에 맞춰 패키지 명 수정 후 머지하겠습니다.